src/index.coffee.md

```diff
@@ -53,6 +58,18 @@ Stream API, for maximum of power:
           callback null, chunks.join ''
       stringifier
 
+You can also use *util.promisify* native function (Node.js 8+) in order to wrap callbacks into promises for more convenient use when source is a readable stream and you are OK with storing entire result set in memory:
+
+    const {promisify} = require('util');
```

Comment:
This exemple should be wrapped inside triple tild "```", otherwise, the exemple will be interpreted by CoffeeScript literate as part of the source code below.

Reply:
Updated.